covalent_dispatcher/_core/execution.py

```diff
@@ -668,25 +726,38 @@ def task_callback(future: Future):
         result_object._status = Result.POSTPROCESSING_FAILED
         result_object._error = f"Post-processing failed: {err}"
         result_object._end_time = datetime.now(timezone.utc)
-        with DispatchDB() as db:
-            db.upsert(result_object.dispatch_id, result_object)
-        result_object.save()
+        result_object.upsert_lattice_data(DispatchDB()._get_data_store())
         result_webhook.send_update(result_object)
 
         return result_object
 
     pp_end_time = post_process_result["end_time"]
     app_log.debug(f"Post-processing completed at {pp_end_time}")
-
     result_object._result = post_process_result["output"]
 
-    # app_log.debug(f"Result: {result_object._result}")
-
+    try:
```

Comment:
Hi @FyzHsn , this block shouldn't be here. This was the old postprocessing step which is now [run in an executor](https://github.com/AgnostiqHQ/covalent/commit/f06f84a0cc487a85ac88177c903f898fe317f845#diff-a99ca6e887d5a6439efcda540e9e2661cdafa9e3a92109f2ac9c390299f794fbL424).

Reply:
Oooh thanks, took this out!